src/palindrom.js

```diff
@@ -622,10 +672,14 @@ var Palindrom = (function() {
         this.network.useWebSocket = newValue;
       }
     });
-
+    /**
+     * how many OT operations are there in each patch 0, 1 or 2
+     */
+    this.OTPatchIndexOffset = 0;
```

Comment:
Or, `this.OTPatchIndexOffset = 1 * options.localVersionPath + 1 * options.remoteVersionPath;`, but yours is definitely more readable :P

Reply:
This shouldn't matter since it runs once.